src/pages/learn.tsx

```diff
@@ -0,0 +1,722 @@
+  <Flex mx={{ base: 0, xl: 36 }} direction="column" gap="0.8rem">
+    {children}
+  </Flex>
+)
+
+const AdditionalReadingHeader = ({ children }: ChildOnlyProp) => (
+  <OldHeading
+    as="h3"
+    mt={16}
+    fontSize="xl"
+    fontWeight="bold"
+    textAlign="center"
+  >
+    {children}
+  </OldHeading>
+)
+
+const Section = ({ children }: ChildOnlyProp) => (
+  <Box as="section" mt={24} _first={{ mt: 0 }}>
+    {children}
+  </Box>
+)
+
+const CardGrid = ({ children }: ChildOnlyProp) => (
+  <Grid
+    templateColumns="repeat(auto-fill, minmax(min(100%, 280px), 1fr))"
+    gap={8}
+    mt={8}
+  >
+    {children}
+  </Grid>
+)
+
+const H2 = ({ children, ...props }: HeadingProps) => (
+  <OldHeading
+    fontSize={{ base: "2xl", md: "2rem" }}
+    lineHeight={1.4}
+    {...props}
+  >
+    {children}
+  </OldHeading>
+)
+
+const H3 = ({ children, ...props }: HeadingProps) => (
+  <OldHeading as="h3" fontSize={{ base: "xl", md: "2xl" }} {...props}>
+    {children}
+  </OldHeading>
+)
+
+export const getStaticProps = (async (context) => {
+  const { locale } = context
+
+  // load i18n required namespaces for the given page
+  const requiredNamespaces = getRequiredNamespacesForPage("/learn")
+  const lastDeployDate = getLastDeployDate()
+
+  return {
+    props: {
+      ...(await serverSideTranslations(locale!, requiredNamespaces)),
+      lastDeployDate,
+    },
+  }
+}) satisfies GetStaticProps<SSRConfig>
+
+const LearnPage = () => {
+  const { t } = useTranslation("page-learn")
+
+  const lgBp = useToken("breakpoints", "lg")
+
+  const tocItems = [
+    {
+      id: "what-is-crypto-ethereum",
+      title: t("toc-what-is-crypto-ethereum"),
+    },
+    {
+      id: "how-do-i-use-ethereum",
+      title: t("toc-how-do-i-use-ethereum"),
+    },
+    {
+      id: "what-is-ethereum-used-for",
+      title: t("toc-what-is-ethereum-used-for"),
+    },
+    {
+      id: "strengthen-the-ethereum-network",
+      title: t("toc-strengthen-the-ethereum-network"),
+    },
+    {
+      id: "learn-about-the-ethereum-protocol",
+      title: t("toc-learn-about-the-ethereum-protocol"),
+    },
+    {
+      id: "learn-about-the-ethereum-community",
+      title: t("toc-learn-about-the-ethereum-community"),
+    },
+    {
+      id: "books-and-podcasts",
+      title: t("toc-books-and-podcasts"),
+    },
+  ]
+  const tocData: ToCItem[] = tocItems.map(({ id, title }) => ({
+    title,
+    url: "#" + id,
+  }))
+
+  const heroContent: CommonHeroProps = {
+    title: t("learn-hub"),
+    header: t("hero-header"),
+    description: t("hero-subtitle"),
+    heroImgSrc: heroImage,
+    buttons: [
+      {
+        content: t("hero-button-lets-get-started"),
+        toId: tocItems[0].id,
+        matomo: {
+          eventCategory: "learn hub hero buttons",
+          eventAction: "click",
+          eventName: "lets get started",
+        },
+        variant: "solid",
+      },
+    ],
+  }
+
+  const height200 = { h: 200, style: { width: "auto" } }
```

Comment:
Next Image only supports `height`.
```suggestion
  const height200 = { height: 200, style: { width: "auto" } }
```

Reply:
You can see here the impact of this change:

current version (will try to fetch the source image)
![image](https://github.com/ethereum/ethereum-org-fork/assets/468158/5ba8caa2-b520-4cba-b1e5-e4d1af9fc74b)

using `height`
![image](https://github.com/ethereum/ethereum-org-fork/assets/468158/51f67c7f-1efd-4ab7-aa95-65cb5796f1ed)

Reply:
Okay thanks... same mistake on /staking page... didn't realize `w` was different than `width` here... will patch